### PR TITLE
Update copy per issue 101

### LINKED ISSUE #101: Update splash page copy
As per https://docs.google.com/document/d/1pjZrPP-OWCCf5C1YMpG6NL1reaMNYrODv6KA602j4ig/edit#heading=h.fv78uyhwladq

### PR DESCRIPTION
**Pull request by iphands**
This fixes #101

### CODE REVIEW

**Comment by iphands** [comment]
I guess to review, use this:

screencapture-prod-foo-redhat-1337-insightsbeta-splash-1496172685806

File: app/js/states/splash/splash.jade [hunk 1]
```diff
@@ -23,6 +23,24 @@
...
+      .col-sm-8.col-md-6
+        //img.logo.logo-ansible(src='../images/l_ansible.svg')
+        h2.section-title(translate) Automate discovery and remediation
+        p(translate) Automate the resolution of Insights findings through Ansible Playbooks.
+        p
+          strong Red Hat Insights&nbsp;
+          span(translate) creates significant time savings and helps avoid firefighting by enabling automatic resolution of critical issues before they affect your business. Optimize your environment and boost security with
+          strong Red Hat Insights.
+
+        a(href='mailto:insights@redhat.com')
```
Comment (Robcwilliams) [inline app/js/states/splash/splash.jade]: Removal of lines 40-42 (email address) & this should be all set

Comment (iphands) [inline app/js/states/splash/splash.jade]: Ninja edit the doc much? :rofl:

Comment (iphands) [inline app/js/states/splash/splash.jade]: tmp

Comment (iphands) [inline app/js/states/splash/splash.jade]: Good ^?

**Pull request merged**